metals/src/main/scala/scala/meta/internal/metals/Compilers.scala

```diff
@@ -150,6 +152,20 @@ class Compilers(
       pc.complete(CompilerOffsetParams.fromPos(pos, token)).asScala
     }.getOrElse(Future.successful(new CompletionList()))
 
+  def autoImports(
+      params: CodeActionParams,
+      name: String,
+      token: CancelToken
+  ): Future[ju.List[AutoImportsResult]] = {
+    val textDocumentPositionParams = new TextDocumentPositionParams(
+      params.getTextDocument(),
+      params.getRange().getEnd()
```

Comment:
code actions are triggered on a range, but the presentation compiler needs a position, so I'm arbitrarily picking the end of the range.